product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -154,6 +154,15 @@ When enabling read-only workloads, keep in mind the following:
     
 For information on replication lag while using read-only workloads, see [Standby replicas](/biganimal/latest/overview/02_high_availability/#standby-replicas). 
 
+#### PgBouncer
+
+!!! Note
+    Enabling PgBouncer incurs additional costs. For more information, see [PgBouncer costs](../../pricing_and_billing/#pgbouncer-costs).
+
+Enable **PgBouncer** to have it manage your connections to Postgres databases and help your workloads run more efficiently. To learn more about PgBouncer, see the [Supported extensions and tools](/biganimal/latest/overview/extensions_tools/#edb-pgbouncer) topic.
+
+Use the **PgBouncer Configuration Settings** dropdown menu to set PgBouncer-specific GUCs. Select the Read-Write and Read-Only tabs according to the type of connection you wish to configure. 
```

Comment:
Let's call these PgBouncer-specific settings, not PgBouncer-specific GUCs. The PgBouncer docs themselves call them settings and not GUCs: https://www.pgbouncer.org/config.html. GUC is a Postgres-specific acronym.